api/serverless.yml

```diff
@@ -26,13 +26,18 @@ functions:
           path: /infected
           method: post
           integration: lambda-proxy
-  getDevicesNearInfected:
-    handler: handler.getDevicesNearInfected
+  getDeviceRisk:
```

Comment:
Nice, how did you like the framework?

Reply:
it's great, scheduling driven by yaml is very simple and straightforward

Reply:
yep, usage of all type of events is very neat